### DISCUSSION TITLE
macOS Guests: Update to the latest version not possible

### DISCUSSION THREAD

**Discussion by tbussmann** [utmapp/UTM#5287] [macOS]
I do have virtualized Guests running 12.6.3 and 13.2.1 just fine. When I try to install the update to 12.6.5 / 13.3.1 offered in the System Preferences, it downloads and starts to install up to the point where the VM reboots. During that I get a message from UTM "Internal Virtualization error. The virtual machine stopped unexpectedly." and the VM stops. After that I can start it again, but the update is not installed. Previous updates installed w/o a problem in the VMs.

UTM is Version 4.2.5 (81) from GitHub, Host ist MBP with M1 running Monterey 12.6.5.

Is this a known issue or what could I do to provide further information to debug the issue?

**Comment by mdcap** [comment]
I too have observed this: it seems that OS updates are not possible in a UTM virtual machine at all.
**EDIT:** actually, now i can't remember if i updated to 12.6.3 from 12.6. But definitely updating from 12.6.3 doesn't seem possible.


The best 'workaround' i can think of is to avoid updating, and just create a VM running the target OS using the appropriate IPSW file (https://ipsw.me/Mac14,2)

**Comment by MyMattes** [comment]
I was able to successfully update macOS 13 "Ventura" and macOS 14 "Sonoma" using the default procedure via settings. But "Monterey" always boots into an unresponsive black screen while applying the update to macOS 12.6.3 ... will restore the IPSW, now.

__Edit__: The macOS 12.6.1 IPSW was causing the same effect, while 12.6 installed correctly...

**Reply by mdcap** [reply]
That's my experience too; updating macOS Ventura works perfectly 👍

**Comment by impala-sc** [comment]
seems to still be a problem with macOS Tahoe 26.6.2 > 26.7 on UTM 4.7.5

**Reply by osy** [reply]
Latest version is 5.0.6 beta
